guidelines/sc/22/focus-appearance.html

```diff
@@ -7,11 +7,11 @@ <h4>Focus Appearance</h4>
     
     <p>When a <a>user interface component</a> has keyboard focus, one or both of the following is true:</p>
     <ol>
-        <li>The <a>focus indicator</a>:
+        <li>The entire <a>focus indicator</a>:
             <ul>
                 <li><a>encloses</a> the visual presentation of the user interface component, and</li>
                 <li>has a contrast ratio of at least 3:1 between the same pixels in the focused and unfocused states, and</li>
-                <li>has a contrast ratio of at least 3:1 against adjacent colors.</li>
+                <li>has a contrast ratio of at least 3:1 against adjacent non-focus-indicator colors.</li>
```

Comment:
I think this is confusing, more than it is helpful.

Reply:
It matches the wording in the second set of bullets and it's pretty important because you can have a situation where you could arguably fail a two-tone focus indicator (or just a non-solid colour) because someone could argues there isn't 3:1 contrast between adjacent colours. Saying "non-focus-indicator" makes it emphatic that one is ONLY assessing this bullet against something adjacent to the focus indicator, not pixels within the indicator itself.

Reply:
@mbgower isn't that undone by starting off the bullets with "entire"? 
If you are looking at the entire focus indicator, you have to take the edges of that for adjacent, even if it creates contrast within itself by having light & dark areas?
Either way it's going to fall through to the measures bullets.

Reply:
Oh, for the two-tone, you're right. Some of that is going to fail (by design) so you would be assessing the area criteria.